### ISSUE TITLE
Support color coding executables, symbolic links, and empty files

### ISSUE THREAD

**Issue by Brixy** [sxyazi/yazi#262]
Currently yazi supports color coding by `mime` and `name`.

[As discussed](https://github.com/sxyazi/yazi/pull/161#issuecomment-1758092119) I would like to suggest to support color coding

- executable files,
- symbolic links, and
- empty files (or maybe different colors for different file sizes, including 0 byte)

Color codes for these cases should (probably) overwrite `mime` and `name` color settings.

_Personally_ I just use the following setting (light theme) and would consider color coding the cases mentioned above to be more important.

```
[filetype]
rules = [
	{ name = "*", fg = "#000000" },
	{ name = "*/", fg = "#0000ee" }
]
```

Thank you very much for considering this—and congratulation for your massive pr [#161](https://github.com/sxyazi/yazi/pull/161).

**Comment by sxyazi** [comment]
Thanks for the detailed report. This issue will be fixed in Yazi v1.0.6. Related issue: https://github.com/sxyazi/yazi/issues/129

**Comment by sxyazi** [comment]
> empty files (or maybe different colors for different file sizes, including 0 byte)

This is already supported, you can try:

```
{ mime = "inode/x-empty", fg = "your color" }
```
